test-harness.sh

```diff
@@ -50,7 +55,11 @@ echo "$THIS: seconds it took to get to end of cloning and copying: $(($(date "+%
 
 ## Start test harness environment
 pushd "$SDK_TESTING_HARNESS"
-./scripts/up.sh
+
+[[ "$VERBOSE_HARNESS" = 1 ]] && V_FLAG="-v" || V_FLAG=""
```

Comment:
nit: what is this line for, checking `VERBOSE_HARNESS` match with `V_FLAG`, is that correct?

Reply:
Nahhh... This is bash'es version of the ternary operator. In java this would be:

```java
boolean vFlag = verboseHarness == 1 ? "-v" : "";
```

Reply:
(might need parentheses in Java...)

Reply:
holy... the `&&` forms a ternary operator...